jobs/promptLicenceCreation.ts

```diff
@@ -58,9 +59,11 @@ const buildEmailGroups = async (
       if (!config.rollout.probationAreas.includes(responsibleCom.probationArea.code)) {
         return null
       }
+      const prisonerWithCRN = prisoner.nomisRecord
+      prisonerWithCRN.crn = prisoner.deliusRecord.offenderCrn
```

Comment:
meh.. can do it all in one assignment and remove the mutation:
`const prisonerWithCRN = { ...prisoner.nomisRecord, crn: prisoner.deliusRecord.offenderCrn }`